src/components/accounts/RecoverAccountSeedPhrase.js

```diff
@@ -55,7 +56,7 @@ class RecoverAccountSeedPhrase extends Component {
         }
 
         const { accountId, seedPhrase } = this.state
-        await this.props.recoverAccountSeedPhrase(seedPhrase, accountId)
+        await this.props.recoverAccountSeedPhrase(seedPhrase, false, accountId, true)
```

Comment:
nit: what does `false` and `true` mean here? best to either store them in local variables first (so it's clear) or pack into options objects so everything is named.